src/pages/settings/Preferences/PreferencesPage.tsx

```diff
@@ -94,28 +89,28 @@ function PreferencesPage(props) {
                                 <View style={[styles.flex1, styles.alignItemsEnd]}>
                                     <Switch
                                         accessibilityLabel={translate('preferencesPage.muteAllSounds')}
-                                        isOn={lodashGet(props.user, 'isMutedAllSounds', false)}
+                                        isOn={user?.isMutedAllSounds ?? false}
                                         onToggle={User.setMuteAllSounds}
                                     />
                                 </View>
                             </View>
                             <MenuItemWithTopDescription
                                 shouldShowRightIcon
-                                title={translate(`priorityModePage.priorityModes.${props.priorityMode}.label`)}
+                                title={translate(`priorityModePage.priorityModes.${priorityMode ?? CONST.PRIORITY_MODE.DEFAULT}.label`)}
                                 description={translate('priorityModePage.priorityMode')}
                                 onPress={() => Navigation.navigate(ROUTES.SETTINGS_PRIORITY_MODE)}
                                 wrapperStyle={styles.sectionMenuItemTopDescription}
                             />
                             <MenuItemWithTopDescription
                                 shouldShowRightIcon
-                                title={translate(`languagePage.languages.${preferredLocale}.label`)}
+                                title={translate(`languagePage.languages.${LocaleUtils.getLanguageFromLocale(preferredLocale)}.label`)}
                                 description={translate('languagePage.language')}
                                 onPress={() => Navigation.navigate(ROUTES.SETTINGS_LANGUAGE)}
                                 wrapperStyle={styles.sectionMenuItemTopDescription}
                             />
                             <MenuItemWithTopDescription
                                 shouldShowRightIcon
-                                title={translate(`themePage.themes.${props.preferredTheme || CONST.THEME.DEFAULT}.label`)}
+                                title={translate(`themePage.themes.${preferredTheme ?? CONST.THEME.DEFAULT}.label`)}
```

Comment:
I that possible that `preferredTheme` is an empty string? If yes, the pipes (`||`) are preferred to.

Reply:
According to ThemePage logic, the preferredTheme prop can only take values from CONST.THEME, so I don't think it should be a problem